src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/book-store
-spring.datasource.username=root
-spring.datasource.password=Bs1234!
+spring.datasource.username=
+spring.datasource.password=
```

Comment:
Why you clear credentials for DB?

Reply:
Because I got a comment before, that it is a sensitive data, that must not be in PRs.